rosbot_description/urdf/wheel.urdf.xacro

```diff
@@ -60,16 +84,28 @@
       </collision>
 
       <inertial>
-        <mass value="0.100" />
-        <inertia ixx="0.00007" ixy="0.0"       ixz="0.0"
-                               iyy="0.0001263" iyz="0.0"
-                                               izz="0.00007" />
+        <mass value="${mass}" />
+        <inertia ixx="${ixx}" ixy="0.0"    ixz="0.0"
+                              iyy="${iyy}" iyz="0.0"
+                                           izz="${izz}" />
       </inertial>
     </link>
 
-    <gazebo reference="${prefix}_wheel_link">
-      <material>Gazebo/DarkGrey</material>
-    </gazebo>
+    <xacro:if value="${use_sim}">
+      <gazebo reference="${prefix}_wheel_link" xmlns:ignition="http://ignitionrobotics.org/schema">
+        <collision>
+          <surface>
+            <friction>
+              <ode>
+                <mu>1.0</mu>
+                <mu2>0.0</mu2>
+                <fdir1 ignition:expressed_in="base_link">${fdir}</fdir1>
+              </ode>
+            </friction>
+          </surface>
+        </collision>
+      </gazebo>
+    </xacro:if>
```

Comment:
This does not have to be guarded by `use_sim` param. Thos are not standard URDF tags, so they will not be parsed outside of Gazebo